tests/006-test-dns.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+job_name=dns
+
+test_dns_nomad_job() {
+    pushd ~/go/src/github.com/Roblox/nomad-driver-containerd/example
+
+    nomad_major_version=$(nomad --version|sed -n 's/Nomad v\([0-9]\+\).*/\1/p')
+    if [ "$nomad_major_version" -le 0 ];then
```

Comment:
If we are moving to nomad version `1.0.2` in `run_tests.sh`. Do we still need this check?

Reply:
removed.